src/components/header/SubHeader.astro

```diff
@@ -75,7 +75,7 @@ const shaking = version === 'dev' ? '3deg' : '1deg';
             !isDsl2 ? (
                 <div class="alert alert-danger p-1 p-md-3" role="alert">
                     <span>
-                        This pipeline uses DSL1. It will not work with Nextflow version after <code>22.03.0-edge</code>.{' '}
+                        This pipeline uses DSL1. It will not work with Nextflow versions after <code>22.10.x</code>.{' '}
```

Comment:
```suggestion
                        This pipeline uses DSL1. It will not work with Nextflow versions after <code>22.10.6</code>.{' '}
```
Thanks eager <3